docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: "3"
+
+services:
+  app:
+    build:
+      context: .
+    ports:
+      - "7000:7000"
+    volumes:
+      - ./:/app
+    command: >
+      sh -c "python3 manage.py migrate &&
+            python3 manage.py wait_for_db &&
+            python3 manage.py runserver 0.0.0.0:7000"
```

Comment:
interesting, why did you decide to use the `7000` port?

Reply:
Because 8000 is not work on Windows(